pkg/util/tls-reload.go

```diff
@@ -168,7 +168,7 @@ func NewCertReloader(config ReloadConfig) (*CertReloader, error) {
 		return nil, err
 	}
 	// If SIA is not issuing certificates, use pollRefresh function to periodically read certificate files and update cert reloader.
-	if config.ProviderService == "" {
+	if config.ProviderService == "" && config.CertFile != "" && config.KeyFile != "" {
```

Comment:
Please roll back to maintain this PR only focusing on the panic bug.
I made a separate pr for this fix here: https://github.com/AthenZ/k8s-athenz-sia/pull/118
```suggestion
	if config.ProviderService == ""  {
```

Reply:
rollback in https://github.com/AthenZ/k8s-athenz-sia/pull/111/commits/a50f2abc9836b9e82a400416652cd70ba48ef762